test/manipulation.test.js

```diff
@@ -542,6 +543,22 @@ describe('manipulation', function() {
       });
     });
 
+    it('should reject created StubUser with empty password', function(done) {
+      StubUser.create({email: 'b@example.com', password: ''}, function(err, createdUser) {
+        (err.message).should.match(/password cannot be empty/);
+        done();
+      });
+    });
+
+    it('should reject updated empty password with updateAttribute', function(done) {
+      StubUser.create({password: 'abc123'}, function(err, createdUser) {
```

Comment:
it is not needed in 3.x, as the user instance is designed without `email` .. as well as it will generate an error since the email is not defined.